builder/fetch.py

```diff
@@ -237,3 +256,30 @@ def fetch_script(url, script_path):
 
     print('Applying exec permissions to {}'.format(script_path))
     os.chmod(script_path, stat.S_IRUSR | stat.S_IWUSR | stat.S_IXUSR)
+
+
+def publish_package(name, package_path):
```

Comment:
minor, but "publish" makes me think maven vs. copy-to-s3

Reply:
What would you prefer? I can update it.

Reply:
copy_package_to_s3 maybe?

Reply:
I feel like that leaks the abstraction a bit. I would like to keep the storage end of this obfuscated, in case it changes to a package manager or something in the future.

Reply:
If we want to change it later, I'm happy to. It's an easy refactor.